client/src/notifications/Notifications.present.jsx

```diff
@@ -323,7 +325,7 @@ class NotificationsMenuList extends Component {
 
     return (
       <Fragment>
-        <Link to="/notifications">
+        <Link to={Url.get(ABSOLUTE_ROUTES.v1.notifications)}>
```

Comment:
Do not use `Url.get()` with `ABSOLUTE_ROUTES`.

```suggestion
        <Link to={ABSOLUTE_ROUTES.v1.notifications}>
```